content/news/2019-12-03-mapstruct-and-quarkus.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "MapStruct and Quarkus - a match made in heaven?"
+author: Christian Bandowski
+date: "2019-12-03"
+tags: [news, examples]
+---
+
+This year is already over, but it was started with something new that came up in the Java world: [Quarkus](https://quarkus.io/). You may already have heard about it, if not, don’t worry, I will quickly summarize what it is.
+
+_tl;dr: Take a look at the README in our [examples repository](https://github.com/mapstruct/mapstruct-examples/tree/master/mapstruct-quarkus)._
+
+<!--more-->
+
+### Quarkus
+> Supersonic Subatomic Java
+> 
+> A Kubernetes Native Java stack tailored for GraalVM & OpenJDK HotSpot, crafted from the best of breed Java libraries and standards
+
+That’s the description on [quarkus.io](https://quarkus.io/), but what does this mean?
+
+With Quarkus you can built microservices or other Java applications using a stack of already existing components like [Eclipse Vert.x](https://vertx.io/), [Hibernate](http://hibernate.org/) or [Eclipse MicroProfile](https://microprofile.io/). It has a container-first philosophy which means that Quarkus sticks all these frameworks together in a way that the resulting application works very well in container environments like [Kubernetes](https://kubernetes.io/) or [OpenShift](https://openshift.io/).
```

Comment:
Typo: `build` instead of `built`.

I don't quite understand what "Quarkus sticks all these frameworks" should mean.

Reply:
perhaps "puts together"?

Reply:
You are right, it is more understandable to me